STANDARDS.md

```diff
@@ -3,13 +3,42 @@
 This document lists down the standards for OpenSearch plugins which can be used by plugin authors.
 
 - [Plugin Standards](#plugin-standards)
+  - [Development Best Practices](#development-best-practices)
   - [Broken Links Checker](#broken-links-checker)
   - [CI Workflows](#ci-workflows)
   - [Code Coverage Report](#code-coverage-report)
   - [License Headers](#license-headers)
   - [Public Documentation](#public-documentation)
   - [Release Notes](#release-notes)  
 
+## Development Best Practices
+
+Follow these best practices while developing any plugin:
+
+#### Separation of Concerns
```

Comment:
These should be `###` and not `####` to nest under the topic above. And use a VSCode plugin to update the TOC?